client/src/components/ImageContainer.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+function ImageContainer({ isLarge, movie }) {
+  const baseUrl = 'https://image.tmdb.org/t/p/original/';
+
+  return (
+    <>
+      <MovieContainer netflixStyle={isLarge}>
+        <ImageStyler
+          netflixStyle={isLarge}
+          key={movie.id}
+          src={`${baseUrl}${isLarge ? movie.poster_path : movie.backdrop_path}`}
+          alt={movie.name}
+        />
+        <Overlay></Overlay>
+        {/*Add to Watchlist icon will be placed in Overlay*/}
```

Comment:
Try to avoid merging commented out code into main :) Remove this line first

Reply:
OK. Included the comment for the review. Will remove now.